README.md

```diff
@@ -67,4 +68,3 @@ each emulated peer will get the same `localStorage`.
 to really test the storage usage of your Webxdc,
```

Comment:
this last note should probably point to the webxdc-dev tool, that is a better emulator supporting per tab/device localStorage

Reply:
i added a line.